OptimizelySDK.Tests/TestData.json

```diff
@@ -589,6 +593,13 @@
 					"key": "string_variable",
 					"type": "string",
 					"defaultValue": "wingardium leviosa"
+				},
```

Comment:
since C# had a problem before with json type, please add
```{
					"defaultValue": "{\"int_var\": 5212, \"boolean_key\": true}",
					"type": "json",
					"id": "1",
					"key": "true_json_var"
				}
``` 
and test

Reply:
Added the test you asked for. So can you please review if this is what you asked for?  [link](https://github.com/optimizely/csharp-sdk/pull/216/commits/4758fbdc3ca38db6481f5e44437fec8f46062de4)